[2022-01-29 12:27:21,384] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2019-02-01T00:00:00+00:00 [queued]>
[2022-01-29 12:27:21,481] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2019-02-01T00:00:00+00:00 [queued]>
[2022-01-29 12:27:21,481] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-29 12:27:21,482] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-01-29 12:27:21,497] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-29 12:27:21,589] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-02-01 00:00:00+00:00
[2022-01-29 12:27:21,667] {standard_task_runner.py:52} INFO - Started process 3392 to run task
[2022-01-29 12:27:21,734] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'local_to_gcs_task', 'scheduled__2019-02-01T00:00:00+00:00', '--job-id', '254', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmpf20l7zro', '--error-file', '/tmp/tmpfg4stogq']
[2022-01-29 12:27:21,735] {standard_task_runner.py:77} INFO - Job 254: Subtask local_to_gcs_task
[2022-01-29 12:27:22,182] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2019-02-01T00:00:00+00:00 [running]> on host 85e1117cfb12
[2022-01-29 12:27:22,559] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-02-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-02-01T00:00:00+00:00
[2022-01-29 12:27:24,046] {python.py:175} INFO - Done. Returned value was: None
[2022-01-29 12:27:24,435] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=local_to_gcs_task, execution_date=20190201T000000, start_date=20220129T122721, end_date=20220129T122724
[2022-01-29 12:27:24,588] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-29 12:27:25,688] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
